lib/interface/cli/codefresh

```diff
@@ -1,70 +1,75 @@
 #!/usr/bin/env node
 
-const path = require('path');
-const _ = require('lodash');
-const yargs = require('yargs');
-const recursive = require('recursive-readdir');
-const CFError = require('cf-errors');
-const DEFAULTS = require('./defaults');
-const authManager = require('../../logic').auth.manager;
-const { printError } = require('./helpers/general');
+if (process.argv.includes('--get-yargs-completions')) {
+    const initCompletion = require('./completion');
+    initCompletion().argv;
+} else {
+    const _ = require('lodash');
+    const yargs = require('yargs');
+    const path = require('path');
+    const recursive = require('recursive-readdir');
+    const CFError = require('cf-errors');
+    const DEFAULTS = require('./defaults');
+    const authManager = require('../../logic').auth.manager;
+    const { printError } = require('./helpers/general');
 
-process.on('uncaughtException', function (err) {
-    printError(err);
-    process.exit(1);
-});
+    process.on('uncaughtException', function (err) {
```

Comment:
lets put these so that they will also affect the completion part.
in case of any unexpected error.